src/lib/wallet/GoodWalletProvider.js

```diff
@@ -76,8 +76,6 @@ export const GoodWalletProvider = ({ children, disableLoginAndWatch = false }) =
         return [wallet, storage]
       } catch (e) {
         log.error('failed initializing wallet and userstorage:', e.message, e)
-
-        throw e
```

Comment:
please revert this, we need to stop wallet loading on unexpected error
just log then throw as it was before